common/core/nesteddisplay.cpp

```diff
@@ -158,7 +158,9 @@ void NestedDisplay::InitNestedDisplay(uint32_t width, uint32_t height,
 #endif
 }
 
-bool NestedDisplay::Initialize(NativeBufferHandler * /*buffer_handler*/) {
+bool NestedDisplay::Initialize(NativeBufferHandler * /*buffer_handler*/,
+                               FrameBufferManager *frame_buffer_manager) {
+  fb_manager_ = frame_buffer_manager;
```

Comment:
nit picking, instead of this u can use initializer list.
bool NestedDisplay::Initialize(NativeBufferHandler * /*buffer_handler*/,
                                              FrameBufferManager *frame_buffer_manager) : fb_manager_(frame_buffer_manager)

same other places as well.

Reply:
This line wouldn't work since initializer lists are only taken from constructors though right? This, in turn, would change the structure pipeline that the fb_manager gets passed through since some of the constructors for these classes and their corresponding initializer class are called at different times/areas throughout hwc and have init override dependencies like wsr/physicialdisplay, etc.

Reply:
ya sorry for the noise.